src/plugins/newsfeed/public/lib/api.ts

```diff
@@ -63,34 +79,36 @@ export class NewsfeedApiDriver {
   }
 
   updateLastFetch() {
-    sessionStorage.setItem(NEWSFEED_LAST_FETCH_STORAGE_KEY, Date.now().toString());
+    sessionStorage.setItem(this.lastFetchStorageKey, Date.now().toString());
   }
 
   updateHashes(items: NewsfeedItem[]): { previous: string[]; current: string[] } {
     // replace localStorage hashes with new hashes
-    const stored: string | null = localStorage.getItem(NEWSFEED_HASH_SET_STORAGE_KEY);
+    const stored: string | null = localStorage.getItem(this.hashSetStorageKey);
     let old: string[] = [];
     if (stored != null) {
       old = stored.split(',');
     }
 
     const newHashes = items.map((i) => i.hash);
     const updatedHashes = [...new Set(old.concat(newHashes))];
-    localStorage.setItem(NEWSFEED_HASH_SET_STORAGE_KEY, updatedHashes.join(','));
+    localStorage.setItem(this.hashSetStorageKey, updatedHashes.join(','));
 
     return { previous: old, current: updatedHashes };
   }
 
-  fetchNewsfeedItems(http: HttpSetup, config: ApiConfig): Rx.Observable<FetchResult> {
+  fetchNewsfeedItems(http: HttpSetup, config: ApiConfig): Rx.Observable<void | FetchResult> {
```

Comment:
why `void`?

Reply:
Hmm, I'm not sure why I added this. I'll remove it